ister_test.py

```diff
@@ -2023,7 +2089,7 @@ def get_template_good():
     try:
         template = ister.get_template("file://{0}/test.json"
                                       .format(os.getcwd()))
-        if template != {"test": 1}:
+        if template != {"test": 1, "SoftwareManager": "swupd"}:
```

Comment:
Mind adding another test here for dnf?

Reply:
This tests whether when missing a SoftwareManager key that it gets added and defaults to swupd.  Not sure if it makes sense to have a dnf test for this?

Reply:
Ah sorry, I was thinking of tests for validating that dnf and swupd are okay to use.

Reply:
I think I might have that covered with copy_os_switch_xxx (dnf and swupd)?

Reply:
Ah yes indeed.